### ISSUE TITLE
Tile Entities stop working after dimension changes

### ISSUE THREAD

**Issue by Flextt** [AllTheMods/ATM3-Remix#106]
Version: ATM3 Remix 1.0.1, SP, no changes at all to the modlist or config, via MultiMC.

## Expected Behavior
JourneyMap teleports and RFTools Chargable Porter / Matter Transmitter should teleport the player without issue.

## Current Behavior
Occasionally, tile entities break down when not using Vanilla teleport methods. They can not be broken or reconfigured. I observed this issue also in ATM3 but this was tagged as non-reproduceable. To prevent world corruption or ID conflicts, I made sure to keep the state of the modpack as "out of the box".

## Possible Solution
Save & Quit and reloading the world.

## Steps to Reproduce
1. Use one of the aforementioned teleport methods a few times
2. Check tile entities

## Log
Link to the [latest.log and another crash report from vanillafix from RFTools](https://gist.github.com/Flextt/9d12afdcd4967740d381c62642ae77ce)

**Comment by Yamza** [comment]
What tile entities are you referring to here? Just the Waystone or something else? I have noticed the Waystone not being able to teleport before. Likely an issue with the mod and not something we can fix.

**Comment by Flextt** [comment]
I dont really get the question. I dont use Waystones. Anyway, both vanilla and modded tile entities become completely unuseable.

**Comment by Yamza** [comment]
What tile entities?

**Comment by Flextt** [comment]
Beds, the aforementioned teleporters, RFTools stuff, Thermal Expansion stuff, chests... None of them can be broken or have their properties changed after this occurs.

**Comment by Yamza** [comment]
Would potentially be helpful to see your latest log, you can upload it to gist. https://gist.github.com/

**Comment by Flextt** [comment]
Link to the [latest.log and another crash report from vanillafix from RFTools](https://gist.github.com/Flextt/9d12afdcd4967740d381c62642ae77ce)

**Comment by Yamza** [comment]
Have you tried anything to attempt to fix this? The error report seems to indicate something not working properly, but not specific enough to say what. 

I would recommend reinstalling the pack fresh (also there is an update to remix), making a new world there and attempting to recreate the issue.

**Comment by FallenEagle1** [comment]
Before reinstalling, I would remove vanillafix and see if the problem persists.

**Comment by Flextt** [comment]
For the next test, I disabled vanillafix and optifine and used a backup from before the RFTools crash. Then I did the same things as before and lo and behold, tile entities stop working. Then I ported back to The End which surprisingly was still updating and used the Endportal to come back. The Overworld was then ticking again. The log shows this starting at 10:47. I tried rebuilding the RFTools Matter Transmitter / Receiver but that didnt resolve the problem. [Log of the test run.](https://gist.github.com/Flextt/5164f284b0333aa788a1a7482d9c6090)
